### PR TITLE
Include more lyrics

### PR DESCRIPTION
**Pull request by Leasw144**
I've included more lyrics to the song!!! Out of order!!!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,16 @@
 # merge-conflict-excercise
-This is a practice exercise for git merge conflics
 
-What's the worst that I can say?
-Things are better if I stay
-So long and goodnight
-So long not goodnight
+I look around and Sin City's cold and empty (oh)
```
Comment (Atos20) [inline README.md]: I like this line very much and it is so deep !!

File: README.md [hunk 2]
```diff
@@ -1,7 +1,16 @@
...
+I can't see clearly when you're gone
+
+I'm just walking by to let you know (by to let you know)
+I can never say it on the phone (say it on the phone)
+Will never let you go this time (ooh)
+I said, ooh, I'm blinded by the lights
+No, I can't sleep until I feel your touch
+Hey, hey, hey
+
+I'm running out of time
```
Comment (Atos20) [inline README.md:17]: me too!! lol

File: README.md (hunk 1 above)
Comment (Atos20) [reply README.md]: could you change the last work also!

**Comment by Leasw144** [comment]
I changed the word empy for moist.

**Pull request merged**